netbox/resource_netbox_aggregate.go

```diff
@@ -83,11 +83,14 @@ func resourceNetboxAggregateRead(d *schema.ResourceData, m interface{}) error {
 
 	res, err := api.Ipam.IpamAggregatesRead(params, nil)
 	if err != nil {
-		errorcode := err.(*ipam.IpamAggregatesReadDefault).Code()
-		if errorcode == 404 {
-			// If the ID is updated to blank, this tells Terraform the resource no longer exists (maybe it was destroyed out of band). Just like the destroy callback, the Read function should gracefully handle this case. https://www.terraform.io/docs/extend/writing-custom-providers.html
-			d.SetId("")
-			return nil
+		if errresp, ok := err.(*ipam.IpamAggregatesReadDefault); ok {
+			errorcode := errresp.Code()
+			if errorcode == 404 {
+				// If the ID is updated to blank, this tells Terraform the resource no longer exists (maybe it was destroyed out of band). Just like the destroy callback, the Read function should gracefully handle this case. https://www.terraform.io/docs/extend/writing-custom-providers.html
+				d.SetId("")
+				return nil
+			}
+
```

Comment:
Can you get rid of this space? (everywhere)

Reply:
Thanks, I thought I addressed that, but apparently not. Removed the extra whitespace everywhere.